src/Uno.UI.Runtime.Skia.WebAssembly.Browser/BrowserNativeElementHostingExtension.cs

```diff
@@ -11,7 +11,7 @@ namespace Uno.UI.Runtime.Skia;
 internal partial class BrowserNativeElementHostingExtension : ContentPresenter.INativeElementHostingExtension
 {
 	private readonly ContentPresenter _presenter;
-	private static string _lastSvgClipPath = "";
+	private static string? _lastSvgClipPath;
```

Comment:
On first render we did not create the svg file with the clippath because the default value was the same we are sending when skpath was empty.